common/tests/conftest.py

```diff
@@ -33,6 +33,13 @@ def staff_api_client():
     return _create_api_client_with_user(UserFactory(is_staff=True))
 
 
+@pytest.fixture
+def guardian_api_client():
+    return _create_api_client_with_user(
+        UserFactory(is_staff=True, guardian=GuardianFactory())
```

Comment:
does it need `is_stuff=True`?

Reply:
No, and should not have it either, good catch 👍